x/cdp/keeper/keeper.go

```diff
@@ -110,3 +110,23 @@ func (k Keeper) IterateCdpsByCollateralRatio(ctx sdk.Context, collateralType str
 
 	}
 }
+
+// SetSavingsRateDistributed sets the SavingsRateDistributed in the store
+func (k Keeper) SetSavingsRateDistributed(ctx sdk.Context, totalDistributed sdk.Int) {
+	store := prefix.NewStore(ctx.KVStore(k.key), types.SavingsRateDistributedKey)
+	bz := k.cdc.MustMarshalBinaryLengthPrefixed(totalDistributed)
+	store.Set(types.SavingsRateDistributedKey, bz)
```

Comment:
this is actually storing the key under `types.SavingsRateDistributedKey + types.SavingsRateDistributedKey` (i.e duplicated). Use `[]byte{}` instead
```suggestion
	store.Set([]byte{}, bz)
```

Reply:
https://github.com/Kava-Labs/kava/pull/644/commits/2ffcdfa8feb3fb5916bc0eb63d91669d1f65f6ba